README.md

```diff
@@ -134,15 +136,16 @@ The [track anatomy documentation](https://github.com/exercism/docs/blob/master/l
         │       │       ├── package.yaml
         │       │       └── src
         │       │           └── Accumulate.hs
+        |       ├── config.json
         │       └── hints.md
         ├── allergies
         │   └── ...
         └── ...
 ```
 - `config.json`: Every exercise has to be registered here. It has a unique name and a difficulty. The sequence order is also the default order in which the exercises are fetched.
 
-## Exercise structure
-Each exercise has the following structure:
+## Concept exercise structure
```

Comment:
the list of files below should include a descripion of config.json and .docs/instructions.md.